package.json

```diff
@@ -11,7 +11,8 @@
     "dev": "ts-node index.ts",
     "lint": "eslint . --ext .ts",
     "lint-fix": "eslint . --ext .ts --fix",
-    "prod": "tsc -p . && node bin/index.js"
+    "build": "tsc -p .",
+    "prod": "yarn build && node bin/index.js"
```

Comment:
Separating out the build step from `prod` to allow for only checking if the build passes before a deploy